src/commands/distribute/groups/update.ts

```diff
@@ -159,28 +171,30 @@ export default class UpdateDistributionGroupCommand extends AppCommand {
     }
   }
 
-  private async removeTestersFromDistributionGroup(client: MobileCenterClient, app: DefaultApp, userEmails: string[]): Promise<models.DistributionGroupUserDeleteResponse[]> {
+  private async deleteTestersFromDistributionGroup(client: MobileCenterClient, app: DefaultApp, userEmails: string[]): Promise<string[]> {
     try {
-      const httpResponse = await out.progress("Removing testers from the distribution group...", 
+      const httpResponse = await out.progress("Deleting testers from the distribution group...", 
         clientRequest<models.DistributionGroupUserDeleteResponse[]>((cb) => client.account.deleteDistributionGroupUsers(app.ownerName, app.appName, this.distributionGroup, {
           userEmails
         }, cb)));
       if (httpResponse.response.statusCode >= 400) {
         throw httpResponse.response.statusCode;
       } else {
-        return httpResponse.result;
+        // TODO: replace once result swagger is fixed
+        // return httpResponse.result.filter((result) => result.status < 400).map((result) => result.userEmail);
+        return (<any[]> JSON.parse((<any> httpResponse.response).body)).filter((result) => result.status < 400).map((result) => result.user_email);
```

Comment:
There is a problem with the swagger for deleteDistributionGroupUsers operation. It specifies the single object DistributionGroupUserDeleteResponse as a result, but the operation actually returns an array of DistributionGroupUserDeleteResponse objects. 
I have implemented this workaround for now. Once the bug will be fixed, it will be replaced with the previous line.